scripts/jams_to_mirex_pattern.py

```diff
@@ -0,0 +1,65 @@
+
+import csv
+import json
+
+x = open('/path/to/file.jams').read()
+x = json.loads(x)
+
+record = []
+
+for x in x["annotations"]:
```

Comment:
It's best to not re-use variable names like this: `x` takes on at least 3 different meanings in the current code, making it difficult to read.  It's better to do `for annotation in x['annotations']`.

Additionally, if you use `jams.load()` instead of raw json, you can use the jams search logic to narrow down to only the annotations in pattern discovery namespace:

```
jam = jams.load('file.jams')
for annotation in jam.annotations['pattern']:
    output each annotation...
```

Reply:
yes that was dangerous sorry. 
same issue about using jams.load #196